osu.Game/Overlays/Settings/SettingsFooter.cs

```diff
@@ -67,6 +71,17 @@ private void load(OsuGameBase game, OsuColour colours, RulesetStore rulesets)
                     Colour = DebugUtils.IsDebug ? colours.Red : Color4.White,
                 },
             };
+
+            this.game = game;
```

Comment:
Would prefer you use the `[Resolved]` attribute for these instead.

Reply:
I chose to fetch dependencies from load() because some of them (`OsuGameBase`) was already being used in the load() method.